src/components/axes/categoryAxis.ts

```diff
@@ -146,15 +146,19 @@ export module Axis {
           if (this._tickLabelAngle !== 0) { // rotated label
             width = axisHeight - this._maxLabelTickLength() - this.tickLabelPadding(); // use the axis height
           }
+          // HACKHACK: Wrapper fails under negative circumstances
+          width = Math.max(width, 0);
         }
 
         // HACKHACK: https://github.com/palantir/svg-typewriter/issues/25
         var height = bandWidth; // default for left/right
         if (this._isHorizontal()) { // case for top/bottom
-          height = axisHeight;
+          height = axisHeight - this._maxLabelTickLength() - this.tickLabelPadding();
           if (this._tickLabelAngle !== 0) { // rotated label
             height = axisWidth - this._maxLabelTickLength() - this.tickLabelPadding();
           }
+          // HACKHACK: Wrapper fails under negative circumstances
+          height = Math.max(height, 0);
```

Comment:
Was `height` ever zero before, or is it just due to the change you made above (line 156)?